### ISSUE TITLE
How can I upload more than one file?

### ISSUE THREAD

**Issue by massinlaaouaj** [timvisee/ffsend#66]
Hi Tim I try to send two [or more] files into the command ffsend upload but I can't, if you can help to know what is the command for uploading two or more files, please?

Exmpl;  ffsend upload /home/usuari/Desktop/all-in-one.png /home/usuari/Desktop/all-in-one.sh

**Comment by timvisee** [comment]
Currently, this isn't possible. Though, I'd like to have support for it. I never came to implementing this.

You can currently upload a single directory (which can contain any number of files), which is automatically archived. You might be able to use this as workaround.

Manually archiving the files first achieves the same:
```bash
tar czvf archive.tar.gz all-in-one.png all-in-one.sh
ffsend upload archive.tar.gz
```

I'm not sure when this feature will be available to support multiple file inputs. I might be able to take a look into it tonight, but I won't promise anything. Thank you for reporting this.

**Comment by timvisee** [comment]
Had some time available. I've implemented first basic support for uploading multiple files, so, does now work:
```bash
ffsend upload my_file another_file ./some_dir/ --name my_archive
```

It archives all the provided files in a similar manner as Firefox Send does. So you don't have to do it yourself anymore.

It doesn't yet provide the list of archived files to Firefox Send, meaning that the download page in your browser does only show the single archived file. This is just a visual thing and doesn't affect anything else. I'll implement that at a later time.

This feature has been implemented in `v0.2.47`, which should be available once the following release pipeline succeeds:
https://gitlab.com/timvisee/ffsend/pipelines/67276182
